cmd/detectExecuteScan.go

```diff
@@ -6,22 +6,22 @@ import (
 
 	"github.com/SAP/jenkins-library/pkg/command"
 	"github.com/SAP/jenkins-library/pkg/log"
+	"github.com/SAP/jenkins-library/pkg/telemetry"
 )
 
-func detectExecuteScan(myDetectExecuteScanOptions detectExecuteScanOptions) error {
+func detectExecuteScan(config detectExecuteScanOptions, telemetryData *telemetry.CustomData) {
```

Comment:
There is a generator available for missing `<stepname>.go` files. This should also be reflected there..

Reply:
do you see anything missing there https://github.com/SAP/jenkins-library/blob/593f1925aba963722d0561b85d917635b8f72e2c/pkg/generator/helper/helper.go#L160?

Reply:
👍